pkg/results/reader.go

```diff
@@ -194,6 +209,10 @@ func ExtractConfig(path string, info os.FileInfo, conf *config.Config) error {
 
 // Functions for helping with backwards compatibility
 
+func (a *Reader) Metadata() string {
```

Comment:
Please add a comment on this public function

Reply:
Hmm that should have been caught by the build jiggery... 
/cc @liztio

Reply:
I think that's part of the linter, which we decided was too flaky to actually use to gate builds?